test/generators/go/__snapshots__/GoGenerator.spec.ts.snap

```diff
@@ -0,0 +1,65 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`GoGenerator generateCompleteModels() should render dependencies 1`] = `
+"
+package some_package
+import (  
```

Comment:
I'm confused why there is no new line between `package` declaration and before/after `import` one, but there is here: https://github.com/asyncapi/modelina/pull/540/files#diff-b5e5cf8b3a4b39ae9bd7669e4a81ccb851cdc9802b1615e7162dc568f4c35381R39

Reply:
The reason it renders the new line is that a string as set here: https://github.com/asyncapi/modelina/blob/c58c71c8c4749dd06e8b81dab784c88a5f28480f/src/generators/go/GoGenerator.ts#L92

Renders the actual new lines contained with the ```

Reply:
Intersting. So then it means all those lines should be separated by an empty line: https://github.com/asyncapi/modelina/blob/c58c71c8c4749dd06e8b81dab784c88a5f28480f/src/generators/go/GoGenerator.ts#L93-L95

Reply:
If you want to have an empty line between them then yes. At the moment, the empty line is present because there was no imports 🙂

Reply:
I mean, it is not strictly necessary because the program will run anyway but it looks unaesthetic if there is no empty line between package and import, import and rest. But nothing else so maybe we can just skip it for this time.